<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>400</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color :'white'</string>
   </property>
   <widget class="QLabel" name="InputItemName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color : rgb(118, 173, 232);
border-radius :10px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Update Item &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>910</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton::active {
background-color : rgb(177, 220, 242);
border-radius : 25px;
}
QPushButton::hover{
background-color : rgb(31, 204, 181);
}

QPushButton::pressed{
background-color : rgb(30, 115, 227);
}


</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color : rgb(177, 220, 242);
border-radius : 15px;
}</string>
    </property>
    <property name="text">
     <string>Item Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color : rgb(177, 220, 242);
border-radius : 15px;
}</string>
    </property>
    <property name="text">
     <string>Item Quantity</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color : rgb(177, 220, 242);
border-radius : 15px;
}</string>
    </property>
    <property name="text">
     <string>Item Price</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveUpdatePriceQty">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton::active {
background-color :  rgb(16, 166, 235);
border-radius : 25px;
}
QPushButton::hover{
background-color : rgb(31, 204, 181);
}

QPushButton::pressed{
background-color : rgb(30, 115, 227);
}</string>
    </property>
    <property name="text">
     <string>Update Price/Quantity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toMainMenu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton::active {
background-color :  rgb(16, 166, 235);
border-radius : 25px;
}
QPushButton::hover{
background-color : rgb(31, 204, 181);
}

QPushButton::pressed{
background-color : rgb(30, 115, 227);
}</string>
    </property>
    <property name="text">
     <string>Back to Main Menu</string>
    </property>
   </widget>
   <widget class="QLabel" name="Status">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {

border-radius :10px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxItemQty">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxItemPrice">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color : rgb(177, 220, 242);
border-radius : 15px;
}</string>
    </property>
    <property name="text">
     <string>Item Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveUpdateName">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton::active {
background-color :  rgb(16, 166, 235);
border-radius : 25px;
}
QPushButton::hover{
background-color : rgb(31, 204, 181);
}

QPushButton::pressed{
background-color : rgb(30, 115, 227);
}</string>
    </property>
    <property name="text">
     <string>Update Nama</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color : rgb(177, 220, 242);
border-radius : 15px;
}</string>
    </property>
    <property name="text">
     <string>Item Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableView" name="ItemTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView {
border-style : none
}</string>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ItemNameBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
margin-left:10px;

}
QLineEdit::active {
background-color : rgb(118, 173, 232);
border-radius : 15px;
border-style: none;
}
QLineEdit::hover{
background-color : rgb(31, 204, 181);
}

QLineEdit::pressed{
background-color : rgb(30, 115, 227);
}


</string>
    </property>
    <property name="placeholderText">
     <string>Nama Barang</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ItemNameBoxUpdate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
margin-left:10px;

}
QLineEdit::active {
background-color : rgb(118, 173, 232);
border-radius : 15px;
border-style: none;
}
QLineEdit::hover{
background-color : rgb(31, 204, 181);
}

QLineEdit::pressed{
background-color : rgb(30, 115, 227);
}


</string>
    </property>
    <property name="placeholderText">
     <string>Update Nama Brarang</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ItemNamePriceQtyBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
margin-left:10px;

}
QLineEdit::active {
background-color : rgb(118, 173, 232);
border-radius : 15px;
border-style: none;
}
QLineEdit::hover{
background-color : rgb(31, 204, 181);
}

QLineEdit::pressed{
background-color : rgb(30, 115, 227);
}


</string>
    </property>
    <property name="placeholderText">
     <string>Nama Barang</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ItemQtyBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
margin-left:10px;

}
QLineEdit::active {
background-color : rgb(118, 173, 232);
border-radius : 15px;
border-style: none;
}
QLineEdit::hover{
background-color : rgb(31, 204, 181);
}

QLineEdit::pressed{
background-color : rgb(30, 115, 227);
}


</string>
    </property>
    <property name="placeholderText">
     <string>Update Jumlah</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ItemPriceBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
margin-left:10px;

}
QLineEdit::active {
background-color : rgb(118, 173, 232);
border-radius : 15px;
border-style: none;
}
QLineEdit::hover{
background-color : rgb(31, 204, 181);
}

QLineEdit::pressed{
background-color : rgb(30, 115, 227);
}


</string>
    </property>
    <property name="placeholderText">
     <string>Update Harga</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cart</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
